apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.js

```diff
@@ -8,6 +8,99 @@ window.PLOrderBlocks = function (uuid, options) {
 
   let optionsElementId = '#order-blocks-options-' + uuid;
   let dropzoneElementId = '#order-blocks-dropzone-' + uuid;
+  let pressedKeys = {};
```

Comment:
having a global dictionary for this is a bit messy, can't we just use this instead? https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/shiftKey